src/main/java/pl/simpleascoding/tutoringplatform/advertisement/Advertisement.java

```diff
@@ -17,6 +17,7 @@
 @Table(name = "advertisement")
 @NoArgsConstructor
 public class Advertisement {
+
```

Comment:
entity shouldn't be public

Reply:
That will be reworked in further commits, probably there will be separate task for that as reducing entity accessibility is linked with part of the code rework ( some of them are tight coupled with other services than their own )